src/test/java/pageObjects/HomePage.java

```diff
@@ -0,0 +1,21 @@
+package pageObjects;
+
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage {
```

Comment:
Create abstract `BasePage` and keep there `driver` in `protected` field. Then use it as parent class for your pages. Hence you will be able to use `driver` in any page if needed.

Reply:
Done